api/envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.proto

```diff
@@ -16,6 +16,10 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#extension: envoy.filters.http.grpc_json_transcoder]
 
 // [#next-free-field: 10]
+// GrpcJsonTranscoder filter configuration.
+// The filter itself can be used per route / per virtual host or on the general level. The most
+// specific one is being used for a given route. If the list of services is empty - filter
+// is considered to be disabled.
```

Comment:
We need to be more descriptive on per-route configuration, because it is really confusing is grpc transocder is changing route decision, that means the per-route configuration for grpc-transcoder must match the route BEFORE grpc transcoder, as opposed to what actually transocded requests are routed to.